js/foundation.util.mediaQuery.js

```diff
@@ -105,6 +113,16 @@ var MediaQuery = {
     this._watcher();
   },
 
+  /**
+   * Reinitializes the media query helper what might be necessary if your CSS is not (completely) loaded while _init() is executed.
+   * @function
+   * @private
+   */
+  _reinit() {
+    this.isInitialized = false;
+    this._init();
```

Comment:
This would populate the `queries` array again.
You could remove `queries: []` and set up `queries` in the `_init()` method.